src/rx5808-pro-diversity/rx5808-pro-diversity.ino

```diff
@@ -122,6 +122,9 @@ screens drawScreen;
 
 #endif
 
+void setupPins();
+void setupSettings();
+void haltWithError();
```

Comment:
Thanks for making these explicit. I keep forgetting the Arduino IDE does a bunch of weird stuff to "help" you.